QUICK_START_GUIDE.md

```diff
@@ -11,175 +11,286 @@
-
-- **🍎 Apple M1/M2/M3 Mac or 🐧 Linux system** (tested on Fedora). We anticipate support for more operating systems in the future.
-- C++ compiler
-- Python 3.9+
-- Approximately 60GB disk space (entire process)
-
 ## ✅ Getting started
 
 ### 🧰 Installing `ilab`
 
-1. If you are on Fedora Linux, install C++, Python 3.9+, and other necessary tools by running the following command:
+1. When installing on Fedora Linux, install C++, Python 3.9+, and other necessary tools by running the following command:
+
+   ```shell
+   sudo dnf install g++ gcc make pip python3 python3-devel python3-GitPython
+   ```
 
-    ```bash
-    sudo dnf install g++ gcc make pip python3 python3-devel python3-GitPython
-    ```
+   Optional: If `g++` is not found, try `gcc-c++` by running the following command:
 
-   Optional: If g++ is not found, try 'gcc-c++' by running the following command:
+   ```shell
+   sudo dnf install gcc-c++ gcc make pip python3 python3-devel python3-GitPython
+   ```
 
-     ```bash
-     sudo dnf install gcc-c++ gcc make pip python3 python3-devel python3-GitPython
-     ```
+   If you are running on macOS, this installation is not necessary and you can begin your process with the following step.  
 
-2. Create a new directory called `instructlab` to store the files the `ilab` CLI needs when running and CD into the directory by running the following command:
+2. Create a new directory called `instructlab` to store the files the `ilab` CLI needs when running and `cd` into the directory by running the following command:
 
-   ```bash
+   ```shell
    mkdir instructlab
    cd instructlab
    ```
 
-   > **NOTE:** The following steps in this document use [Python venv](https://docs.python.org/3/library/venv.html) for virtual environments. However, if you use another tool such as [pyenv](https://github.com/pyenv/pyenv) or [Miniforge](https://github.com/conda-forge/miniforge) for managing Python environments on your machine continue to use that tool instead. Otherwise, you may have issues with packages that are installed but not found in `venv`.
+   > **NOTE:** The following steps in this document use [Python venv](https://docs.python.org/3/library/venv.html) for virtual environments. However, if you use another tool such as [pyenv](https://github.com/pyenv/pyenv) or [Conda Miniforge](https://github.com/conda-forge/miniforge) for managing Python environments on your machine continue to use that tool instead. Otherwise, you may have issues with packages that are installed but not found in `venv`.
 
 3. Install and activate your `venv` environment by running the following command:
 
-   ```bash
-   python3 -m venv venv
+   > **NOTE**: ⏳ `pip install` may take some time, depending on your internet connection. In case installation fails with error ``unsupported instruction `vpdpbusd'``, append `-C cmake.args="-DLLAMA_NATIVE=off"` to `pip install` command.
+
+   See [the GPU acceleration documentation](./docs/gpu-acceleration.md) for how to
```

Comment:
We didn't want to "link off this page." The idea is that this is the "no fluff" way to get `ilab` working, for labs or workshops and the like. This stuff is for the main README in `instructlab/instructlab`